doc/input-output-reference/src/overview/group-airflow.tex

```diff
@@ -44,11 +44,9 @@ \subsubsection{Inputs}\label{inputs-005}
 
 The name of the ZoneInfiltration:DesignFlowRate object. This needs to be unique across all different ZoneInfiltration objects.
 
-\paragraph{Field: Zone or ZoneList Name}\label{field-zone-or-zonelist-name}
+\paragraph{Field: Zone or ZoneList or Space or SpaceList Name}\label{field-zone-or-zonelist-name}
 
-This field is the name of the zone (ref: Zone) or \hyperref[zonelist]{ZoneList} (ref: ZoneLIst) and attaches a particular infiltration statement to a thermal zone or set of thermal zones in the building. When the \hyperref[zonelist]{ZoneList} option is used then this infiltration definition is applied to each of the zones in the zone list effecting a global definition for the amount of infiltration flow rate in the zone. The Zonelist option can be used effectively with the flow/area, flow/exteriorarea, flow/exteriorwallarea, or airchanges/hour of the Design Flow Rate Calculation Method.
-
-The name of the actual infiltration object becomes \textless{}Zone Name\textgreater{} \textless{}ZoneInfiltration:DesignFlowRate Object Name\textgreater{} and should be less than the standard length (100 characters) for a name field. If it is greater than this standard length, it may be difficult to specify in output reporting as it will be truncated. A warning will be shown if the generated name is greater than 100 characters. If it duplicates another such concatenated name, there will be a severe error and terminate the run.
+This field applies this ZoneInfiltration:DesignFlowRate object to one or more thermal zones or spaces in the building. The Zone, Zonelist or SpaceList options can be used effectively with the watts/area and watts/person options of the Design Level Calculation Method to place a varying equipment load at the same density in each zone or space. The names of the actual ZoneInfiltration:DesignFlowRate objects may be concatenated as \textless{}Space Name\textgreater{} \textless{}ZoneInfiltration:DesignFlowRate Object Name\textgreater{}. See \hyperref[specifying-applicable-zones-or-spaces]{Specifying Applicable Zone(s) or Space(s)} for more details.
 
```

Comment:
Should be just Zonelist or Spacelist ... The Zonelist or SpaceList options can be used effectively ... to place a varying equipment load at the same density in each zone or space.

Should we add a sentence(s) as: ZoneInfiltration:DesignFlowRate objects used in spaces are additive to ZoneInfiltration:DesignFlowRate objects used for the enclosing zone. For example, a zone with multiple spaces could include a space infiltration object to model additional infiltration for that space over and above the zone level infiltration. or some such wording?

Reply:
> Should be just Zonelist or Spacelist ... The Zonelist or SpaceList options can be used effectively ... to place a varying equipment load at the same density in each zone or space.

Zone is included here, because if the zone has more than one Space, the object will expand and have concatenated names.

> 
> Should we add a sentence(s) as: ZoneInfiltration:DesignFlowRate objects used in spaces are additive to ZoneInfiltration:DesignFlowRate objects used for the enclosing zone. For example, a zone with multiple spaces could include a space infiltration object to model additional infiltration for that space over and above the zone level infiltration. or some such wording?

Maybe, I'll think about it. The reference to "Specifying Applicable Zone(s). . . ." is supposed to cover that. But maybe not enough.